README.md

```diff
@@ -33,6 +33,7 @@ addon | version | summary
 [date_range](date_range/) | 10.0.1.0.0 | Manage all kind of date range
 [dbfilter_from_header](dbfilter_from_header/) | 10.0.1.0.0 | Filter databases with HTTP headers
 [disable_odoo_online](disable_odoo_online/) | 10.0.1.0.0 | Remove odoo.com Bindings
+[mass_editing](mass_editing/) | 10.0.1.0.0 | Mass Editing
```

Comment:
Got it, thank you @moylop260 
I just made a new push removing this change.

Best Regards